arkane/isodesmic.py

```diff
@@ -392,6 +392,28 @@ def multiple_error_canceling_reaction_search(self, n_reactions_max=20, milp_soft
 
         return reaction_list
 
+    def calculate_target_enthalpy(self, n_reactions_max=20, milp_software='lpsolve'):
+        """
+        Perform a multiple error canceling reactions search and calculate hf298 for the target species by taking the
+        median hf298 value from among the error canceling reactions found
+
+        Args:
+            n_reactions_max (int, optional): The maximum number of found reactions that will returned, after which no
+                further searching will occur even if there are possible subsets left in the queue.
+            milp_software (str, optional): 'lpsolve' (default) or 'pyomo'. lpsolve is usually faster.
+
+        Returns:
+            ScalarQuantity: Standard heat of formation at 298 K calculated for the target species
+
+        """
+        reaction_list = self.multiple_error_canceling_reaction_search(n_reactions_max, milp_software)
+        h298_list = np.zeros(len(reaction_list))
+
+        for i, rxn in enumerate(reaction_list):
+            h298_list[i] = rxn.calculate_target_thermo().value_si
+
+        return ScalarQuantity(np.median(h298_list), 'J/mol')
+
```

Comment:
median seems sensible, although it would be nice if we could incorporate some uncertainty for each reaction and put greater weight on reactions with lower uncertainty (uncertainty related to the known high-level hf of the reference species in the reaction or the low-level dft calc)